src/plugins/home/server/services/sample_data/data_sets/otel/index.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { i18n } from '@osd/i18n';
+import path from 'path';
+import { AppLinkSchema, SampleDatasetSchema } from '../../lib/sample_dataset_registry_types';
+import {
+  appendDataSourceId,
+  getSavedObjectsWithDataSource,
+  overwriteSavedObjectsWithWorkspaceId,
+} from '../util';
+// import { getSavedObjects } from './saved_objects';
+import { servicesFieldMappings } from './services_field_mappings';
+import { tracesFieldMappings } from './traces_field_mappings';
+
+const otelDataName = i18n.translate('home.sampleData.otelSpecTitle', {
+  defaultMessage: 'Sample Open-Telemetry logs, Traces and Metrics',
+});
+const otelDataDescription = i18n.translate('home.sampleData.otelSpecDescription', {
+  defaultMessage:
+    'Sample data including correlated observability signals for an e-commerce application in Open-Telemetry standard.',
+});
+const initialAppLinks = [] as AppLinkSchema[];
+
+const DEFAULT_INDEX = 'i254a9691-8a1b-11ef-adb4-9f1097200c20';
```

Comment:
What is this default index from? Seems we won't add any saved objects after sample data imported.

Reply:
Yup, this is not needed anymore. Updated here: [c6949c4](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8587/commits/c6949c4d965ee3542887cbe64baeb84361a8616d)